cime/scripts/lib/CIME/XML/pio.py

```diff
@@ -31,6 +35,15 @@ def get_defaults(self, grid=None, compset=None, mach=None, compiler=None, mpilib
         for node in self.get_children("entry"):
             value = self.get_default_value(node, attributes)
             if value:
-                defaults[self.get(node, "id")] = value
+                myid = self.get(node, "id")
+                iscompvar = myid.split("_")[0] != "PIO"
```

Comment:
@jedwards4b , this seemed like a pretty weak way to check if the current node is component specific, but I could not see any other way to achieve this. Please comment if you know of a better way.

Reply:
For compvars you can have PIO_STRIDE_LND=4 or LND_PIO_STRIDE=4 and both will do the same thing.  This seems to break that capability and only allow PIO_ at the beginning of the name - use env_base.py check_if_compvar instead.  I think that it will also tell you whether this is specific to a component or a general setting.